modules/ROOT/pages/export-to-exchange-task.adoc

```diff
@@ -3,37 +3,71 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio allows you to connect to Exchange to publish your project as a new asset.
+Anypoint Studio enables you to write documentation for your new projects and to connect to Exchange to publish your project as a new asset.
+
+== Publishing Assets with Documentation
+
+When you create a new project in Anypoint Studio, Studio includes an *exchange-docs* folder with a *home.md* file. You can document the front page of your new asset in Markdown format after publishing your asset to Exchange.
+
+NOTE: Learn more about Markdown at https://www.markdownguide.org/.
+
+Studio enables you to add more pages, images, and an icon: 
+
+. Create a new project in Anypoint Studio.
+. In the Package Explorer view, open *Exchange-docs* > *Home.md*.
+. Use the default Studio text editor or choose an external editor by right-clicking on *Open with...* > *Other* and selecting an external app.
+. Optionally, create additional pages to add to your asset documentation by right-clicking on the *Exchange-docs* folder and selecting *New* > *File*.
++
+NOTE: The name of the file will be the name of the page in Exchange.
++
+. If you have more than two pages, create a `config.json` file using the following format  to provide a page order:
++
+----
+{"pageOrder": ["home","page1","page2"]}
+----
++
+If no format is provided, the home page will be followed by the other pages in alphabetical order. 
++
+. Assign an icon to your project by adding an image (*.jpg, *.png, *.bmp) with the name icon before publishing your project for the first time.
+. Create a *Resources* folder in the *Exchange-docs* folder to add extra resources to your documentation, such as images, or other types of files (size limit of 2MB).
+. To publish your project, right-click on your project in the *Package Explorer*, and click on *Anypoint Platform* > *Publish to Exchange*.
```

Comment:
You did not bold Package Explorer view previously (and you also used the term view).

Reply:
Sorry! I got confused! Is it OK to bold "Package Explorer" as part of the UI and not bold "view"? Does it make sense to you?

Reply:
Yes! That's the tab label. Thanks!